app/views/families/index.html.erb

```diff
@@ -1,6 +1,28 @@
-<p class="title"><%= t(".title") %></p>
-<%= link_to t(".add_family"), new_family_path, class: "link_button, button" %>
-<table class="list-table">
-  <%= render "families_table_header" %>
-  <%= render @families %>
-</table>
+<%= render "sub_header"%>
```

Comment:
Since this sub_header is only used in one place, a suggestion to roll the markup back into this view

Reply:
Done in sha `88c5afe`